tests/framework/statistics/types.py

```diff
@@ -26,147 +15,51 @@ class MeasurementDef:
 
     name: str
     unit: str
+    statistics: List['StatisticDef']
 
     @classmethod
-    def cpu_utilization_vmm(cls):
-        """Return vmm cpu utilization measurement definition."""
-        return MeasurementDef(
-            DefaultMeasurement.CPU_UTILIZATION_VMM.name.lower(),
-            "percentage"
-        )
+    def create_measurement(cls,
```

Comment:
Unfortunately, dataclasses have already a predefined constructor [1], mapped on the fields declared above, so I can not provide a constructor with similar signature, since python classes do not accept multiple constructors.

[1] https://docs.python.org/3/library/dataclasses.html